com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
+    "rename-icon": "trenchbroom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--filesystem=host",
+        "--filesystem=home"
```

Comment:
The `--filesystem=host` privileges already contain `--filesystem=home` permissions.

Reply:
Oops. Probably a left-over when I attempted to restrict the access. Fixed it.